launch/dynamo-run/src/lib.rs

```diff
@@ -194,6 +195,15 @@ pub async fn run(
     #[cfg(any(feature = "vllm", feature = "sglang"))]
     let mut extra: Option<Pin<Box<dyn Future<Output = ()> + Send>>> = None; // vllm and sglang sub-process
 
+    let template = if let Some(path) = flags.request_template.as_ref() {
+        Some(RequestTemplate::load(path)?)
+    } else {
+        None
+    };
+    if let Some(template) = template.as_ref() {
+        tracing::info!("Using request template: {template:?}");
+    }
```

Comment:
Could you move this up to line 199 so that we can remove the `if`?

Reply:
And change it to debug before we merge.

Reply:
Sure, will update this.